scripts/upgrade

```diff
@@ -75,24 +33,12 @@ then
 	ynh_script_progression --message="Upgrading source files..." --weight=1
 
 	# Download, check integrity, uncompress and patch the source from app.src
-	ynh_setup_source --dest_dir="$final_path" --keep=".env"
+	ynh_setup_source --dest_dir="$install_dir" --keep=".env nc" --full_replace
```

Comment:
Somehow this does not clean out the old NodeJS stuff (`Dockerfile .gitignore index.js noco-daily.js node_modules .npm package.json package-lock.json README.md .env.example`), so the webapp gets confused and still prompts for an upgrade even though it is up-to-date.

Reply:
Yes, =1 was missing from --full_replace. Fixed.